README.md

```diff
@@ -1,7 +1,5 @@
 # electrino
 
-:warning: **Do not use this in production. This project is in its early stages** :warning:
```

Comment:
I would favor keeping the warning with only the second sentence, maybe with some extra info. Idea:

```
:warning: __This project is in its early stages and contributions would be much appreciated.__ :warning:
```

Reply:
Good idea - let's be explicit about what we need help with. "Port more APIs from Electron into this project"?

Reply:
Please check now.